gluster-exporter/metric_volume_counts.go

```diff
@@ -9,6 +9,10 @@ import (
 
 var (
 	volumeLabels = []MetricLabel{
+		{
```

Comment:
this repetition can be avoided.

```
var clusterIDLabel = MetricLabel{
    Name: "cluster_id",
    Help: "Cluster ID",   
}
```

and use it as

```
volumeLabls = []MetricLabel{
    custerIDLabel,
    {
...
```

Reply:
Ack